tests/ui/option_if_let_else.fixed

```diff
@@ -97,7 +97,7 @@ enum DummyEnum {
     Two,
 }
 
-// should not warn since there is a compled complex subpat
+// should not warn since there is a complex subpat
```

Comment:
i think that `compled` was a typo for `complex` and then `complex` was added again -- that only a single `complex` was intended. But I could be wrong.